src/lib/chips/chip-list.ts

```diff
@@ -274,12 +274,17 @@ export class MatChipList extends _MatChipListMixinBase implements MatFormFieldCo
   @Input('aria-orientation') ariaOrientation: 'horizontal' | 'vertical' = 'horizontal';
 
   /**
-   * Whether or not this chip is selectable. When a chip is not selectable,
+   * Whether or not this chips are selectable. When a chip is not selectable,
    * its selected state is always ignored.
    */
   @Input()
   get selectable(): boolean { return this._selectable; }
-  set selectable(value: boolean) { this._selectable = coerceBooleanProperty(value); }
+  set selectable(value: boolean) { 
+    this._selectable = coerceBooleanProperty(value);
+    if (this.chips) {
+      this.chips.toArray().forEach(chip => chip.chipListSelectable = this._selectable);
```

Comment:
IIRC the `QueryList` already has a `forEach` method so you shouldn't need the `toArray` call.